src/templates/dev_pipeline.handlebars

```diff
@@ -1,3 +1,5 @@
+<input type="text" class="search-thing">
```

Comment:
I'll let @MariagraziaAlastra address this but you might want to call `search-thing` something more specific :palm_tree: